expandable-impl/src/expansion.rs

```diff
@@ -90,7 +91,7 @@ where
         tree: &TokenTree<Span>,
     ) -> Result<Set<(DynamicState<Span>, Transition, Id)>, Error<Span>>
     where
-        Id: Clone + Eq + Hash,
+        Id: Clone + Eq + Hash + Ord,
```

Comment:
Remove hash-related trait bounds.

Side note: they should be kept for the `rust-grammar-dpdfa` crate, as this may be useful to end users.